easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.2-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('freetype', '2.7.1'),
```

Comment:
@boegel But why am I not getting the issue on our local machines?  Is the use-existing-modules disguising it?

Should I fix this and force-rebuild locally?